ethexe/ethereum/src/lib.rs

```diff
@@ -270,3 +271,36 @@ impl SignerSync for Sender {
         self.chain_id
     }
 }
+
+// TODO: Maybe better to append solution like this to alloy.
+/// Works like `tx.get_receipt().await`, but retries a few times if rpc returns a null response.
+async fn get_transaction_receipt<T: transports::Transport + Clone, N: network::Network>(
```

Comment:
maybe it would be better to create trait with `try_get_receipt` method
`get_transaction_receipt(builder.send().await?).await?` vs `builder.send().await?.try_get_receipt().await?`

Reply:
Cannot see the difference, also I think It is a temporary solution...

Reply:
the code is easier to read and not difficult to implement

Reply:
Done